x/ibc/spec/01_concepts.md

```diff
@@ -6,3 +6,40 @@ order: 1
 
 > NOTE: if you are not familiar with the IBC terminology and concepts, please read
 this [document](https://github.com/cosmos/ics/blob/master/ibc/1_IBC_TERMINOLOGY.md) as prerequisite reading.
+
+### Connection Version Negotation
```

Comment:
can we add a list of supported feature set options?

Reply:
this is TBD to the best of my knowledge. I was under the impression this pr is just for implementing the functionality of version negotiation not actually deciding the semantics to be agreed upon for version 1